server/nodejs/controllers/AliasesService.js

```diff
@@ -74,64 +35,47 @@ exports.aliascount = function(args, res, next) {
       return commonUtils.reportError(res, err);
     }
 
-    console.log('Alias count:', result);
+    console.log('Alias info:', result);
     res.end(JSON.stringify(result));
   };
 
   var arr = varUtils.getArgsArr(argList, args, "GET", cb);
-  syscoinClient.aliasCount.apply(syscoinClient, arr);
+  syscoinClient.aliasInfo.apply(syscoinClient, arr);
 }
 
-exports.aliasfilter = function(args, res, next) {
+exports.aliasnew = function(args, res, next) {
   var argList = [
-    { prop: "regexp" },
-    { prop: "from", defaultValue: "" },
-    { prop: "count", defaultValue: "10" },
-    { prop: "safesearch", defaultValue: "Yes" }
+    { prop: "aliasname" },
+    { prop: "publicvalue" },
+    { prop: "accept_transfers_flags", defaultValue: 3 },
+    { prop: "expire_timestamp", defaultValue: 3600 },
+    { prop: "address"},
+    { prop: "encryption_privatekey"},
+    { prop: "encryption_publickey"},
+    { prop: "witness"}
   ];
 
-  args.count.value = varUtils.correctTypes(args.count.value, varUtils.TYPE_CONVERSION.NUM_TO_STRING);
-
   var cb = function(err, result, resHeaders) {
     res.setHeader('Content-Type', 'application/json');
 
     if (err) {
       return commonUtils.reportError(res, err);
     }
 
-    console.log('Alias filter:', result);
-    res.end(JSON.stringify(result));
-  };
-
-  var arr = varUtils.getArgsArr(argList, args, "GET", cb);
-  syscoinClient.aliasFilter.apply(syscoinClient, arr);
-}
-
-exports.aliashistory = function(args, res, next) {
-  var argList = [
-    { prop: "aliasname" }
-  ];
-
-  var cb = function(err, result, resHeaders) {
-    res.setHeader('Content-Type', 'application/json');
-
-    if (err) {
-      //TODO: fix after b1
-      return res.end(JSON.stringify(err.toString()));
-      //return commonUtils.reportError(res, err);
-    }
-
-    console.log('Alias history:', result);
+    console.log('Alias new:', result);
     res.end(JSON.stringify(result));
   };
 
-  var arr = varUtils.getArgsArr(argList, args, "GET", cb);
-  syscoinClient.aliasHistory.apply(syscoinClient, arr);
+  var arr = varUtils.getArgsArr(argList, args, "POST", cb);
+  syscoinClient.aliasNew.apply(syscoinClient, arr);
 }
 
-exports.aliasinfo = function(args, res, next) {
+exports.aliaspay = function(args, res, next) {
   var argList = [
-    { prop: "aliasname" }
```

Comment:
I'm probably missing something, but if I'm paying from an alias, who am I paying *to*?

Reply:
this is actually the `aliasinfo` API call.  which only require the alias you want info of.

Reply:
View of file says:
exports.aliaspay = function(args, res, next) {
  var argList = [
    { prop: "aliasfrom" },
    { prop: "amounts" },
    { prop: "instantsend"},
    { prop: "subtractfeefromamount"}
  ];

Am I reading the wrong thing?